lib/net.c

```diff
@@ -54,6 +52,23 @@ uint8_t default_rss_key[GATEKEEPER_RSS_KEY_LEN] = {
 	0x6a, 0x42, 0xb7, 0x3b, 0xbe, 0xac, 0x01, 0xfa,
 };
 
+
+/* Function to randomize the default RSS key */
+void randomize_rss_key(){
+	srand(time(NULL));
```

Comment:
We are building Gatekeeper within the DPDK framework, so we follow DPDK's coding style:
http://dpdk.org/doc/guides/contributing/coding_style.html

Please format all your pull request to match this coding style.